### PR TITLE
change print to log

### PR DESCRIPTION
**Pull request by BusyJay**
So everything will go to log file.

### CODE REVIEW

**Comment by siddontang** [comment]
LGTM

**Comment by siddontang** [comment]
Must the user initialized the log now?

File: src/lib.rs [hunk 1]
```diff
@@ -242,8 +244,8 @@ impl FailPoint {
                 None => panic!("failpoint {} panic", name),
             },
             Task::Print(msg) => match msg {
-                Some(ref msg) => println!("{}", msg),
-                None => println!("failpoint {} executed.", name),
+                Some(ref msg) => info!("{}", msg),
```
Comment (overvenus) [inline src/lib.rs]: We may lost `msg` if we does not set a logger.

**Comment by BusyJay** [comment]
No if print is not used.

**Comment by siddontang** [comment]
LGTM

**Comment by siddontang** [comment]
PTAL @overvenus

**Comment by BusyJay** [comment]
ping

**Pull request merged**